src/server/models/user.js

```diff
@@ -3,41 +3,31 @@ const mongoose = require('mongoose')
 const uniqueValidator = require('mongoose-unique-validator')
 const Schema = mongoose.Schema
 
-const requiredParamDesc = 'This param is required'
-const duplicateKeyErrorDesc = 'There was a duplicate key'
+const REQUIRED_PARAM_DESC = 'This param is required'
+const DUPLICATE_KEY_ERROR_DESC = 'There was a duplicate key'
 
 
 const userSchema = new Schema({
 	username:{ 
 		type: String,
-		required: requiredParamDesc,
+		required: REQUIRED_PARAM_DESC,
 		unique: true
 	},
 	email:{ 
 		type: String,
-		required: requiredParamDesc,
+		required: REQUIRED_PARAM_DESC,
```

Comment:
Isn't it property for boolean values?

Reply:
nope, it's ok. Instead of boolean true you can put text. This change default error text